compiler/src/dotty/tools/dotc/transform/InstrumentCoverage.scala

```diff
@@ -237,8 +237,8 @@ class InstrumentCoverage extends MacroTransform with IdentityDenotTransformer:
             val InstrumentedParts(pre, coverageCall, expr) = tryInstrument(fun)
 
             if coverageCall.isEmpty then
-              // `fun` cannot be instrumented, and `args` is a type so we keep this tree as it is
-              tree
+            // `fun` cannot be instrumented, and `args` is a type but the expr may have been transformed
```

Comment:
```suggestion
            // `fun` cannot be instrumented and `args` is a type, but `expr` may have been transformed
```

Reply:
Thank you!

Yeah sure I'll see if I can do those things this afternoon.